app/src/main/java/com/example/parkingsystem/activity/ParkingActivity.java

```diff
@@ -23,6 +24,10 @@ protected void onCreate(Bundle savedInstanceState) {
         presenter = new ParkingPresenter(new ParkingModel(), new ParkingView(this, binding));
 
         binding.buttonMainSetAmountParkingSpaces.setOnClickListener(view -> presenter.inflateDialog(this));
+        binding.buttonMainReservation.setOnClickListener(v -> {
+            Intent intent = new Intent(ParkingActivity.this, ReservationActivity.class);
```

Comment:
this is not fine here.. you should follow the MVP structure

Reply:
Fixed